airbyte-integrations/connectors/source-github/source_github/streams.py

```diff
@@ -344,6 +411,18 @@ def path(self, stream_slice: Mapping[str, Any] = None, **kwargs) -> str:
         return f"orgs/{owner}/teams"
 
 
+class Users(Organizations):
```

Comment:
Please overwrite `transform` function for this class and save `organization` field in each record.

Reply:
Fixed